bin/romcal

```diff
@@ -1,3 +1,7 @@
 #!/usr/bin/env node
-var romcal = require('index.js');
-romcal.calendarFor(process.argv[2]);
+var romcal = require('../index.js');
```

Comment:
@jarosz, what does this change? I think @emagnier should review this.

Reply:
This shouldn't be here :)